test/security-council-mgmt/governors/TopNomineesGas.t.sol

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "forge-std/Test.sol";
+import "../../../src/security-council-mgmt/governors/SecurityCouncilMemberElectionGovernor.sol";
+import "../../../src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol";
+
+import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+
+/// @notice This contract tests gas usage of SecurityCouncilMemberElectionGovernor.topNominees()
+contract TopNomineesGasTest is Test {
+    SecurityCouncilMemberElectionGovernor memberGov;
+    SecurityCouncilNomineeElectionGovernor nomineeGov;
+
+    SecurityCouncilNomineeElectionGovernor.InitParams nomineeInitParams =
+    SecurityCouncilNomineeElectionGovernor.InitParams({
+        firstNominationStartDate: Date({year: 2030, month: 1, day: 1, hour: 0}),
+        nomineeVettingDuration: 1 days,
+        nomineeVetter: address(0x11),
+        securityCouncilManager: ISecurityCouncilManager(address(0x22)),
+        securityCouncilMemberElectionGovernor: ISecurityCouncilMemberElectionGovernor(
+            payable(address(0x33))
+            ),
+        token: IVotesUpgradeable(address(0x44)),
+        owner: address(0x55),
+        quorumNumeratorValue: 10000/N,
+        votingPeriod: 1 days
+    });
+
+    struct MemberInitParams {
+        ISecurityCouncilNomineeElectionGovernor nomineeElectionGovernor;
+        ISecurityCouncilManager securityCouncilManager;
+        IVotesUpgradeable token;
+        address owner;
+        uint256 votingPeriod;
+        uint256 maxNominees;
+        uint256 fullWeightDuration;
+    }
+
+    SecurityCouncilMemberElectionGovernor governor;
+
+    uint256 fullWeightDuration = 0.5 days;
+
+    address proxyAdmin = address(0x1111);
+    address voter = address(0x2222);
+
+    uint256 proposalId;
+
+    uint16 constant N = 500;
+
+    function setUp() public {
+        memberGov = SecurityCouncilMemberElectionGovernor(payable(_deployProxy(address(new SecurityCouncilMemberElectionGovernor()))));
+        nomineeGov = SecurityCouncilNomineeElectionGovernor(payable(_deployProxy(address(new SecurityCouncilNomineeElectionGovernor()))));
+
+        // we need to etch code onto each contract parameter
+        _dummyEtch(address(nomineeInitParams.securityCouncilManager));
+        _dummyEtch(address(nomineeInitParams.token));
+        
+        
```

Comment:
Looks like you need to run the formatter

Reply:
ah yes